uiowa/uiowa
# [ITS] Meeting Fixes

Relates to https://github.com/uiowa/uiowa/issues/7510 except for deploying changes out to DEV and setting up email testing.

# To Test

```
ddev blt ds --site=its.uiowa.edu
```

- The support article content type has SEO settings in the sidebar for non-admins. The metatag output should provide basic info and draw from the summary field for description.
- As an non-admin, you can see moderation dropdown options on the service content type edit form. Note: editors do not have the ability to add services.
- As a webmaster, you have the ability to add/edit/delete support articles categories taxonomy.
- Alert node titles should show if they closed pretty much everywhere (card, full node, content overview, alert email) and should update on refresh if the alert ends up closing.
  - https://its.uiowa.ddev.site/admin/content
  - https://its.uiowa.ddev.site/alerts/status-history
  - https://its.prod.drupal.uiowa.edu/alert/956
  - Reminder - To send the alert email, edit the alert that is closed or active and click the boolean at the bottom of the form to send on save. Email should be labelled closed or not depending on the alert you send.
